wordpress-mobile/WordPress-iOS
# Signup Epilogue: don't show updating HUDs if username and password have not changed.

Fixes #9121 

To test:
'Changing username':
1. Sign up for a new account.
2. On the epilogue screen, select username to go to the Change Username screen.
3. Select a username that is not the original.
4. Re-select the original username.
5. 'Back' to return to the epilogue.
6. Press 'Continue'. Verify the 'Changing username' HUD does not appear.

'Changing password':
1. Sign up for a new account via 'Sign up with Email'.
2. On the epilogue screen, tap on the password field.
3. DO NOT enter any text.
4. Tap somewhere to dismiss the keyboard.
5. Press 'Continue'. Verify the 'Changing password' HUD does not appear.
